### PR TITLE
Add testing for is_valid_name function

### PR DESCRIPTION
**Pull request by NicolaasDieBaas**
utils.is_valid_function needs tests to be added

### CODE REVIEW

**Comment by NicolaasDieBaas** [comment]
@gsvr review plz

File: test/ussd_health_worker.test.js [hunk 1]
```diff
@@ -645,6 +646,43 @@ describe("familyconnect health worker app", function() {
...
+                        resultsArray.push(go.utils.is_valid_name(testDataArray[i], 0, 150));
+                    }
+
+                    // expected results
+                    assert.equal(resultsArray.length, 9);
+                    assert.equal(resultsArray[0], true);
+                    assert.equal(resultsArray[1], true);
+                    assert.equal(resultsArray[2], true);
+                    assert.equal(resultsArray[3], true);
+                    assert.equal(resultsArray[4], false);
```
Comment (gsvr) [inline test/ussd_health_worker.test.js]: Please add a comment why it's false (eg. number in J0hn rejected), and add a test for one of the other punctuation marks that should fail.

**Comment by NicolaasDieBaas** [comment]
@gsvr 2nd pass review plz

**Comment by gsvr** [comment]
:+1:

**Pull request merged**